packages/universal-components/src/Picker/PickerTypes.js

```diff
@@ -12,7 +12,11 @@ export type PickerOption = {|
 
 export type Props = {|
   +label?: React.Node,
-  +labelContainerStyle?: StylePropType,
+  /**
+   * Styles for the container and the text of the label
```

Comment:
I believe that this comment is useless, because naming should be (and it is in this case) self-descriptive.

Reply:
Done!